.github/CODEOWNERS

```diff
@@ -0,0 +1,5 @@
+# Lines starting with '#' are comments.
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in the repo.
+*       @voxpupuli/reviewers
```

Comment:
> This CODEOWNERS file contains errors

Looks like there is no @voxpupuli/reviewers

Reply:
we currently don't have a dedicated group for the catalog-diff project, so I think we could use ignore the CODEOWNERS file. our collaborators group can review it by default.

Reply:
File removed